README.md

```diff
@@ -1,4 +1,4 @@
-Calamari is the command-line tool invoked by Tentacle during a deployment. It knows how to extract and install NuGet packages, run the Deploy.ps1 etc. conventions, modify configuration files, and all the other things that happen during an deployment.
+Calamari is the command-line tool invoked by Tentacle during a deployment. It knows how to extract and install NuGet packages, run the build.cmd etc. conventions, modify configuration files, and all the other things that happen during an deployment.
```

Comment:
Hi, this line is incorrect. Deploy.ps1 refers to the scripts it runs at runtime, not the build script